osdk-core/modules/src/filemgr/impl/dji_file_mgr_impl.cpp

```diff
@@ -879,6 +879,7 @@ void FileMgrImpl::fileDataRawDataCB(dji_general_transfer_msg_ack *rsp) {
       (range_handler_->GetLastNotReceiveSeq() == rsp->seq + 1)) {
     DSTATUS("Got the last one pack .");
     packFinishFlag = true;
+    fileDataHandler->mmap_file_buffer_->deInit();
```

Comment:
just a quick question since this looks pretty low-level within the DJI OSDK structure, 
is this to de-initialize a memory-map of some kind, after it's done being utilized.

Reply:
yes, it will release the resources allocated for this buffer as below: 
bool MmapFileBuffer::deInit() {
  DSTATUS("Deinit");
  if (fd >= 0) close(fd);
  fd = -1;
  if (fdAddr) munmap(fdAddr, fdAddrSize);
  fdAddr = NULL;
  return true;
}